### ISSUE TITLE
Power Delivery

### ISSUE THREAD

**Issue by PlanetRageTV** [chrismyers2000/MeshAdventurer#3]
Hey, I really like the design! How is Power Delivery supposed to happen in the V1.3? Only through the Barrel Jack? I would like to power with 16850s or a LiPo Pack, which usually all don't get to the minimum rated input Voltage of 9V. Did you plan for people that want to use it like this to use a stepup voltage converter? (seems rather wasteful :-))

**Comment by chrismyers2000** [comment]
Hi, my future plan was to make one with a solar charger/li-ion charger option but currently I just use a separate board and connect it to the 5v input if I’m using 18650’s. The barrel jack and voltage regulator was just to make it easy to power the device from a 12v outlet in your vehicle. I have kind of abandoned this design as the esp32 is starting to be outdated and is pretty power hungry compared to the nRF units.

**Comment by PlanetRageTV** [comment]
Hey thanks for the feedback! You're using the 5v input of the ESP32 board?

chrismyers2000 ***@***.***> schrieb am Do., 25. Juli 2024,
16:43:

**Comment by chrismyers2000** [comment]
No, there are two small holes to the left of the esp32 that are for 5v. That way the radio has direct access instead of going through the esp32.

**Comment by chrismyers2000** [comment]
But I can tell you when testing the unit while being powered through the esp32 usb port, it seemed to be reliable, granted this depends solely on your 5v supply being able to handle the extra power from the radio, but I do not know if its reliable over the long term.

**Comment by jankowski-t** [comment]
Hey @chrismyers2000, could you release the project files in an editable format aside from the gerbers? I'm making a battery powered version and it would be easier to edit the project rather than recreating it from the schematic and gerbers. I'm specifically looking to keep the board format and placement of key components the same as yours so everything can fit in existing enclosures.
